recipe/meta.yaml

```
@@ -49,18 +45,84 @@ requirements:
     - pytorch
     - pytorch =*=cuda*
     - setuptools
-  run:
-    - einops
-    - python
-    - pytorch =*=cuda*
 
-test:
-  imports:
-    - flash_attn
-  commands:
-    - pip check
-  requires:
-    - pip
+outputs:
+  - name: flash-attn
+    requirements:
+      build:
+        - {{ compiler('cxx') }} # needed for DSO checker
+        - {{ stdlib('c') }}     # needed for DSO checker
```

Think we want all of these so their `run_exports` are applied to this package

```suggestion
      build:
        - {{ compiler('c') }}
        - {{ compiler('cxx') }}
        - {{ compiler('cuda') }}
        - {{ stdlib('c') }}
```